scripts/pow.js

```diff
@@ -19,3 +19,9 @@ function runPow() {
       alert (`n должно быть целым и больше 1`);
   }
 }
+
+
+
+let arr = [2, 5, 3, 9];
+let result = (arr[0]*arr[1]) + (arr[2]*arr[3]);
+alert(result);
```

Comment:
???

Reply:
I solved these tasks as shown in the examples in the book.. very easy)

Reply:
What does this code do in this file?